ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -370,6 +371,10 @@ open class DrawerController: UIViewController {
     }
     private var containerViewCenterObservation: NSKeyValueObservation?
 
+    /// for iPad split mode, navigation bar has a different dark elevated color, and if it is a `.down` presentation style, match `Colors.NavigationBar.background` elevated color
+    private var useNavigationBarBackgroundColor: Bool = false
```

Comment:
is the type needed here?

Reply:
currently the SwiftLint file enforces the type for the property declaration.